src/lib/KeyStore.js

```diff
@@ -20,6 +21,17 @@ class KeyStore {
         return KeyStore._instance;
     }
 
+    /**
+     * @param {KeyRecord} keyRecord
+     * @returns {boolean}
+     */
+    static isEncrypted(keyRecord) {
+        // Because we are supporting legacy secrets which cannot be converted during migration,
+        // a KeyRecord can be both V2 and V3 encrypted.
+        return keyRecord.secret.byteLength === KeyStore.ENCRYPTED_SECRET_SIZE
+            || keyRecord.secret.byteLength === KeyStore.ENCRYPTED_SECRET_SIZE_V2;
```

Comment:
Which is V2 (wasn't that one never used live) and what about V3?

Reply:
In any case it should be documented :)

Reply:
V3 is the new one, V2 is currently used, and V1 was only the default during betanet.